CODEOWNERS

```diff
@@ -2,10 +2,12 @@
 /buildkite @MinaProtocol/infra-eng-reviewers
 /frontend/ @MinaProtocol/product-eng-reviewers
 /dockerfiles/ @MinaProtocol/infra-eng-reviewers
-/scripts/ @MinaProtocol/infra-eng-reviewers @MinaProtocol/protocol-eng-reviewers
+/scripts/ @MinaProtocol/infra-eng-reviewers
 /Makefile @MinaProtocol/infra-eng-reviewers
 /src/config/dev.mlh @MinaProtocol/infra-eng-reviewers @MinaProtocol/protocol-eng-reviewers
 /src/config/testnet_postake_medium_curves.mlh @MinaProtocol/infra-eng-reviewers @MinaProtocol/protocol-eng-reviewers
+/src/config/mainnet.mlh @MinaProtocol/infra-eng-reviewers @MinaProtocol/protocol-eng-reviewers
```

Comment:
Infra should not be able to bypass protocol for setting of the infra constants. Remove infra on here and below.